mirage/factories/authorized-storage-account.ts

```diff
@@ -0,0 +1,28 @@
+import { belongsTo, Factory } from 'ember-cli-mirage';
+import faker from 'faker';
+
+import AuthorizedStorageAccountModel from 'ember-osf-web/models/authorized-storage-account';
+
+import Addons from '../fixtures/addons';
+
+export default Factory.extend<AuthorizedStorageAccountModel>({
+    storageProvider: faker.random.arrayElement(Addons).id,
+    externalUserId: faker.datatype.uuid,
+    externalUserDisplayName: faker.name.findName,
+    scopes: [],
+    defaultRootFolder: faker.system.filePath,
+
+    configuringUser: belongsTo('internal-user'),
```

Comment:
I do not recall doing this for my factories. Is that a thing that needs fixing? What does it do?

Reply:
So this was a misunderstanding of what these do. The [association()](https://www.ember-cli-mirage.com/versions/v0.4.x/factories/#factories-and-relationships) helper can make it so that factories will automatically create a related model (e.g. `server.create('draft-registration')` will automagically create an `initiator` user). Whereas these `belongsTo` and `hasMany` helpers are used to define relationships in your data models (which we don't need to do, since we get this for free as we use ember-data)